### ISSUE TITLE
Issue with "box-shadow": box-shadow examples potentially violate WCAG a11y guidelines with poor colour contrast

### ISSUE THREAD

**Issue by gjdawson** [mdn/content#14372]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

#### What information was incorrect, unhelpful, or incomplete?
Box-shadow offset demonstration has poor colour contrast and is potentially non-compliant with WCAG guidelines. It is particularly hostile to low vision, but otherwise sighted individuals due to poor contrast issues. Poorly contrasting adjacent elements, even if they may be entirely discernable to users with optimal vision, can be difficult or impossible for low-vision users to parse, which may render the rendered example visually confusing and difficult to understand.

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#setting_zero_for_offset_and_blur

#### What did you expect to see?
A good contrast between colours used to demonstrate the feature. While individual developers may stray from WCAG guidelines in their own projects, possibly to their own detriment, a resource such as this, which will be used by low vision developers and developers with restricted colour vision, should comply as strictly as possible in order to maximise accessibility and minimise discrimination against such developers. Replacement of one of the box-shadow colours with black would be the most obvious solution, or replacement of one or both with higher contrast alternatives. I might recommend #2E3436 for the first box-shadow and #EDD400 for the second, as together they are compliant with the guidelines. 

Edit: Alternatively, #1C409B and #F4AAB9 to maintain some resemblance to the previous theme.

#### Did you test this? If so, how?
As well as testing with my own experiences, I used some online a11y resources to quantify the contrast issue. A contrast of less than 3:1 is generally considered non-compliant. The second link demonstrates the issue with poorly contrasted adjacent visual elements (the background and "foreground" border are, in terms of user experience, functionally equivalent to the two box-shadows layered one atop the other).
https://color.a11y.com/ContrastPair/?bgcolor=FF44b9&fgcolor=66CCFF
https://webaim.org/resources/contrastchecker/?fcolor=F4AAB9&bcolor=66CCFF

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/box-shadow`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/box-shadow/index.md
* Last commit: https://github.com/mdn/content/commit/281e3b21178946c8301232a8eb50d11770ee8450
* Document last modified: 2022-02-18T09:53:31.000Z

</details>

**Comment by gjdawson** [comment]
Additionally, as the objection has be raised on the now-closed PR related to this: WCAG 2.1 1.4.11 applies to contrast between adjacent colours, where such colours are necessary to convey meaning or information; the demonstration of box-shadow is intended to convey information to developers, not merely be decorative, which brings it under this rule. The contrast ratio between the colours is well below the threshold the guidelines apply for such circumstances.

**Comment by bsmth** [comment]
Thanks for raising this one @gjdawson. Would you like to make the changes? The source for this page can be found here: https://github.com/mdn/content/blob/main/files/en-us/web/css/box-shadow/index.md?plain=1#L109
